projects/snapshot_bot/snapshot_bot.cc

```diff
@@ -6,7 +6,7 @@
 
 // BoB robotics includes
 #include "common/fsm.h"
-#include "common/logging.h"
+#include "plog/Log.h"
```

Comment:
there's another main deep down here! I wonder if your regex didn't get ones with arguments

Reply:
Hmm it looks like I missed loads, partly because I forgot to commit even the ones I did find :blush: